PeerioTechnologies/peerio-icebear
# chore: new computed hasLegacyObjectsSelected

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/8150/desktop-sdk-disable-shared-folders-as-a-move-destination-for-pending-files
https://github.com/PeerioTechnologies/peerio-desktop/pull/343

#### Testing instructions  
This adds a computed getter that client can use to determine if a legacy file has been bulk selected. See desktop PR for testing instructions.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
